Semibold Author in compact ChatMessages (#20419)
microsoft/fluentui: packages/fluentui/CHANGELOG.md

- Adding brand `backgroundHover3`, brand `backgroundPressed2`, and default `borderTransparentActive` @notandrew ([#20056](https://github.com/microsoft/fluentui/pull/20056))
- Spread event props from trigger in `Popup` @assuncaocharles ([#20162](https://github.com/microsoft/fluentui/pull/20162))
- Fix broken styles debug of undefined `isReactComponent` @petr-duda ([#20349](https://github.com/microsoft/fluentui/pull/20349))

> - Fix font-weight of ChatMessage author slot in compact chat density @Hirse ([#20419](https://github.com/microsoft/fluentui/pull/20419))

Reviewer: ```suggestion
- Use semibold font-weight for ChatMessage author slot in compact chat density @Hirse ([#20419](https://github.com/microsoft/fluentui/pull/20419))
```
